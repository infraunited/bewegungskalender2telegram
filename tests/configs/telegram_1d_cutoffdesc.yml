output:
  type: 'telegram'
  stdout: true
caldav:
  url: 'http://127.0.0.1:18476'
  calendars:
    - url: '/d/701e875a-44ca-97df-0140-bbb36dc26a9b/'
  username: 'x'
  password: 'x'
  offset_days: 0
  number_days: 1
  current_day_override: "2023-01-01"
format:
  time_locale: "de_DE.utf8"
  timezone: "Europe/Berlin"
  show_calendar_name: true
  show_location: true
  show_description: true
  cutoff_description: 20
  header_txt: ""
  footer_txt: ""
