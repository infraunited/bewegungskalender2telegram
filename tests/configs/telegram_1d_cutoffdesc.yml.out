📅 *__Sonntag__ 2023\-01\-01*

__08:30__ *Long timed event* \(testing2\)
🌍 _somewhere_
here is the des\[\.\.\.\]

__09:00__ *Single Event* \(testing2\)